commerce-cloud-code/app_yourshophere/cartridge/experience/components/commerce_layouts/column.json

```diff
@@ -7,9 +7,25 @@
             "id": "additionalStyling",
             "name": "Additional Styling",
             "attribute_definitions": [
+                {
+                    "id": "columnConfiguration",
+                    "name": "Column Configuration",
+                    "type": "custom",
+                    "required": false,
+                    "editor_definition": {
+                        "type": "layout.columnConfigurator"
+                    }
+                },
+                {
+                    "id": "style",
+                    "name": "Style",
+                    "description": "An additional style Class Name",
+                    "type": "string",
+                    "required": false
+                },
                 {
                     "id": "columns",
-                    "name": "Desktop Columns",
```

Comment:
No one uses our app - what do we need to do to get rid of these fields?

Reply:
We need to hit save on all pages. Let me have a look, remove this and see what happens. As I updated pretty much all pages already it might be a non-issue.